src/sentry/buffer/redis.py

```diff
@@ -188,15 +189,20 @@ def incr(self, model, columns, filters, extra=None):
                     frozen_filters = tuple(sorted(filters.items()))
                     key = (frozen_filters, model)
 
-                    stored_columns, stored_extra = _local_buffers.get(key, ({}, None))
+                    stored_columns, stored_extra, stored_exclude_filters = _local_buffers.get(
```

Comment:
dont need stored_exclude_buffers, right? should this be stored_signal_only?

aside does this need to handle both the old value stored and the new value? (2-item tuple vs 3-item)

Reply:
whoops, i missed the default value. fixing this now.